<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="le_output">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_bubble">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_merge">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>120</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Оценить время</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Путь для файла вывода исходного массива:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Путь для файла вывода сортировки пузырьком:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Путь для файла вывода внутренней сортировки:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_hoar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Путь для файла вывода внешней сортировки:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Путь для файла ввода (.pcap/.cap):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_pcap">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ограничение количества пакетов (если 0/full ограничения нет):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_num">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_work">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Working...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_bubble">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bubble time: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hoar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hoar time: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_merge">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Merge time: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>std:sort time: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Путь для файла вывода стандартной сортировки:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_sort">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
